tutorials/intermediate/constrainedlayout_guide.py

```diff
@@ -377,17 +394,17 @@ def docomplicated(suptitle=None):
 # ================================
 #
 # There can be good reasons to manually set an axes position.  A manual call
-# to `ax.set_position()` will set the axes so constrained_layout has no
-# effect on it anymore.  (Note that constrained_layout still leaves the space
-# for the axes that is moved).
+# to `~.axes.Axes.set_position()` will set the axes so constrained_layout has
```

Comment:
see above.

Reply:
I think brackets are allowed. I was questioning the `~.axes` part because I did not realize the wildcard import in `axes`.